ogame/__init__.py

```diff
@@ -370,6 +399,47 @@ class Resources:
 
         return Resources
 
+    def resources_settings(self, id, settings={}):
```

Comment:
settings cant be strings. missspels will fail this function. no ide support. this should be parameters that can be set optionally